README.rst

```diff
@@ -1,11 +1,12 @@
-pep8 - Python style guide checker
+pycodestyle (formerly called pep8) - Python style guide checker
 =================================
 
-pep8 is a tool to check your Python code against some of the style
+pycodestyle is a tool to check your Python code against some of the style
 conventions in `PEP 8`_.
 
 .. _PEP 8: http://www.python.org/dev/peps/pep-0008/
 
+.. note:: This package used to be called :module:`pep8` but was renamed to :module:`pycodestyle` to reduce confusion. Further discussion here: `<https://github.com/PyCQA/pycodestyle/issues/466>`_
```

Comment:
This should be wrapped at ~80 characters and when wrapping you need to do something like

```rst
.. note::

    This package used to be called ``pep8`` but was renamed to ``pycodestyle`` to
    reduce confusion. Further discussion `here 
    <https://github.com/PyCQA/pydocstyle/issues/466>`_.
```

For several reasons:

1. This is the README which is displayed on GitHub and PyPI. Neither will respect the `:module:` since neither uses sphinx
2. Attaching the link to `here` looks nicer.